shared/chat/inbox-search/background.desktop.tsx

```diff
@@ -29,8 +29,7 @@ const Rover = () => {
 
     window.addEventListener('mousemove', onMouseMove, {passive: true})
     return () => {
-      // @ts-ignore mdn suggests we should use the same values as addEventListener
```

Comment:
i tried this manually in devtools and it does correctly unmap it so lets just keep it w/ the type